solana/pyth2wormhole/client/tests/test_set_is_active.rs

```diff
@@ -0,0 +1,119 @@
+pub mod fixtures;
+
+use borsh::BorshDeserialize;
+use p2wc::get_config_account;
+use solana_program_test::*;
+use solana_sdk::{
+    account::Account,
+    pubkey::Pubkey,
+    rent::Rent,
+    signature::Signer,
+    signer::keypair::Keypair,
+};
+
+use pyth2wormhole::config::{
+    P2WConfigAccount,
+    Pyth2WormholeConfig,
+};
+use pyth2wormhole_client as p2wc;
+use solitaire::{
+    processors::seeded::Seeded,
+    AccountState,
+    BorshSerialize,
+};
+
+fn clone_keypair(keypair: &Keypair) -> Keypair {
+    // Unwrap as we are surely copying a keypair and we are in test env.
+    Keypair::from_bytes(keypair.to_bytes().as_ref()).unwrap()
+}
+
+#[tokio::test]
+async fn test_setting_is_active_works() -> Result<(), p2wc::ErrBoxSend> {
+    // Programs
+    let p2w_program_id = Pubkey::new_unique();
+    let wh_fixture_program_id = Pubkey::new_unique();
+
+    // Authorities
+    let p2w_owner = Pubkey::new_unique();
+    let pyth_owner = Pubkey::new_unique();
+    let ops_owner = Keypair::new();
+
+    // On-chain state
+    let p2w_config = Pyth2WormholeConfig {
```

Comment:
for future reference: in other projects, we've had good results extracting out a struct that does all the ProgramTest and account initialization so that you don't have to repeat this code across tests. 

No need to worry about this now though

Reply:
guillermo just made one here for reference https://github.com/pyth-network/pyth-crosschain/pull/292/files

Reply:
Well it depends, as long as it is related to the logic I prefer redundancy because the test itself is more clear. On the case here I think it's better to be as it is right now. Maybe just some helper function to give you, but no magic and no abstraction.